netlink-packet-route/src/rtnl/tc/constants.rs

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT
+
+/// Handles
+pub const TC_H_MAJ_MASK: u32 = 0xFFFF0000;
+pub const TC_H_MIN_MASK: u32 = 0x0000FFFF;
+
+#[macro_export]
+macro_rules! TC_H_MAKE {
+    ($maj: expr, $min: expr) => {
+        ($maj & TC_H_MAJ_MASK) | ($min & TC_H_MIN_MASK)
+    };
+}
+
+pub const TC_H_UNSPEC: u32 = 0;
+pub const TC_H_ROOT: u32 = 0xFFFFFFFF;
+pub const TC_H_INGRESS: u32 = 0xFFFFFFF1;
+pub const TC_H_CLSACT: u32 = TC_H_INGRESS;
+
+pub const TC_H_MIN_PRIORITY: u32 = 0xFFE0;
+pub const TC_H_MIN_INGRESS: u32 = 0xFFF2;
+pub const TC_H_MIN_EGRESS: u32 = 0xFFF3;
+
+/// U32 filters
+pub const TCA_U32_UNSPEC: u16 = 0;
+pub const TCA_U32_CLASSID: u16 = 1;
+pub const TCA_U32_HASH: u16 = 2;
+pub const TCA_U32_LINK: u16 = 3;
+pub const TCA_U32_DIVISOR: u16 = 4;
+pub const TCA_U32_SEL: u16 = 5;
+pub const TCA_U32_POLICE: u16 = 6;
+pub const TCA_U32_ACT: u16 = 7;
+pub const TCA_U32_INDEV: u16 = 8;
+pub const TCA_U32_PCNT: u16 = 9;
+pub const TCA_U32_MARK: u16 = 10;
+pub const TCA_U32_FLAGS: u16 = 11;
+pub const TCA_U32_PAD: u16 = 12;
+pub const __TCA_U32_MAX: u16 = 13;
+pub const TCA_U32_MAX: u16 = __TCA_U32_MAX - 1;
+
+/// U32 Flags
+pub const TC_U32_TERMINAL: u8 = 1;
+pub const TC_U32_OFFSET: u8 = 2;
+pub const TC_U32_VAROFFSET: u8 = 4;
+pub const TC_U32_EAT: u8 = 8;
+pub const TC_U32_MAXDEPTH: u8 = 8;
+
+/// Action attributes
+pub const TCA_ACT_UNSPEC: u16 = 0;
+pub const TCA_ACT_KIND: u16 = 1;
+pub const TCA_ACT_OPTIONS: u16 = 2;
+pub const TCA_ACT_INDEX: u16 = 3;
+pub const TCA_ACT_STATS: u16 = 4;
+pub const TCA_ACT_PAD: u16 = 5;
+pub const TCA_ACT_COOKIE: u16 = 6;
+pub const __TCA_ACT_MAX: u16 = 7;
```

Comment:
I don't think we need this, since we define `TCA_ACT_MAX` just below.

Reply:
fixed